packages/rancher-aks-operator/rancher-aks-operator/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: https://github.com/rancher/aks-operator/releases/download/v1.0.3/rancher-aks-operator-1.0.3.tgz
-version: 100.0.2
+version: 100.0.3
```

Comment:
Bumping these versions is correct, but the versions in the above URL also need to be bumped.

Same for all three operators.